### DISCUSSION TITLE
New, learning, one channel open but can't recieve...

### DISCUSSION THREAD

**Discussion by Rtxa1** [Ride-The-Lightning/RTL#1404] [Q&A] [answered]
I am running RTL over LND. I opened one private channel, and I was able to send SATs out, but now I can't receive.  When I try to receive, the sending app 'cannot find a payment route.' I tried another wallet, and it did not work either.  What can I check?

Thank you for your help.

**Comment by saubyk** [comment, marked as answer]
Hi, to receive payments on a private channel, you need to create an invoice with `Private Routing Hints` turned on.
Also, you should have enough inbound capacity to be able to receive the amount being paid to you.

![image](https://github.com/Ride-The-Lightning/RTL/assets/39208279/1f702696-4da5-4d5e-b943-7647422c5397)

**Reply by Rtxa1** [reply]
That fixed it. Thank you so much.
